### ISSUE TITLE
Please add additional custom ldap filters for users and roles

### ISSUE THREAD

**Issue by kasimon** [rundeck/rundeck#1927]
Hi,

please add the possibility to add additional ldap filters to the login configuration. We need this, because we allow only those users to log into services that have a special attribute set, for example

```
assignedRole: Rundeck
```

Users without this field are supposed not to be able to log in to rundeck. The same with roles, currently all ldap groups a user is member of are used, i want to limit this to ldap groups matching a certain filter.

**Comment by gmangin** [comment]
+1

**Comment by acasademont** [comment]
yes please, this would come in very handy, right now the filter is statically set at `(&(objectClass={0})({1}={2}))`. This string should be made configurable (along with the necessary placeholders) so that we could add another filter.

**Comment by zlanyi** [comment]
Hi Guys,
could you please take care this issue?

Thanks

Cheers,
Zoltan

**Comment by hbermudez-stratio** [comment]
We need this please

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
